pegasus/helpers/section_api_helpers.rb

```diff
@@ -471,6 +476,16 @@ def self.assign_script_to_section(script_id, section_id)
                where(section_id: section_id, deleted_at: nil))
   end
 
+  def self.assign_script_to_user(script_id, user_id)
+    # creates a userscript for a user if they don't have it yet
+    Dashboard.db[:user_scripts].
+      insert_ignore.
+      import([:user_id, :script_id],
+             Dashboard.db[:users].
+               select(user_id, script_id.to_s).
+               where(id: user_id, deleted_at: nil))
```

Comment:
nit: the indentation seems messed up here. I see that it's like that in existing [code](https://github.com/code-dot-org/code-dot-org/blob/d343cd7449e7eb8bdab5ab66e68e455103e103b4/pegasus/helpers/section_api_helpers.rb#L469) so no need to fix.

It might be a bit more readable (and still in line with the [style guide](https://github.com/bbatsov/ruby-style-guide#no-double-indent)) like this?

```ruby
Dashboard.db[:user_scripts].
  insert_ignore.
  import(
    [:user_id, :script_id],
    Dashboard.db[:users].
      select(user_id, script_id.to_s).
      where(id: user_id, deleted_at: nil)
   )
```